generators/languages/templates/src/main/webapp/i18n/bg/metrics.json

```diff
@@ -0,0 +1,102 @@
+{
+    "metrics": {
+        "title": "Метрики на приложението",
+        "refresh.button": "Обновяване",
```

Comment:
Обнови ?!

Reply:
OK, this is a complicated one. I'm not a philologist, so please excuse my humble explanation.
As a source of translation I used English. In English there is (nearly) no difference between "Second Person Singular" and "Second Person Plural". So it is difficult to decide should application provide "Polite" or "Familiar" form of communication.
In Bulgarian language, like in German for example, there is a clear distinction between the two forms (Geman - "du" and "Sie", Bulgarian - Вие" and "ти"). I decided for the "Polite" form. 
And I did my best to keep it consistent all over the messages. This is the way application "talks" to the user.
And here we come to the translation of the buttons. Because more or less these are the activities, that application offers to the user. Or the way user "talks" to the application.
English again has the advantage  - the same word is a Noun and a Verb, even a Verb in its imperative form. But I don't find it nice, if application "talks" in a polite form to put imperative Verbs as button labels. We don't give orders to someone, who is polite to us, do we? 
For this reason I decided to use "Verbal nouns" ("отглаголни съществителни") - Open = "Отваряне", Save = "Запазване", Edit = "Редактиране", and so on.
For this reason, to keep some consistency Refresh = "Обновяване".
I suggest to keep it as it is. Or ... change all buttons to imperative verbs. (Отвори, Изтрий и т.н.)